packages/xsnap/src/xsnap.c

```diff
@@ -880,8 +909,8 @@ void fx_clearTimer(txMachine* the)
 {
 	txJob* job = xsGetHostData(xsArg(0));
 	if (job) {
-        xsForget(job->self);
-        xsSetHostData(xsArg(0), NULL);
+		xsForget(job->self);
```

Comment:
I think this is a gratuitous whitespace change

Reply:
It is not.  I'm normalising the spaces in the file to be tabs.